.gitignore

```
@@ -39,6 +39,7 @@ cata_test-tiles
 cataclysm
 cataclysm-tiles
 cataclysm-tiles.exe.manifest
+cataclysm-bn-tiles.exe.manifest
```

```suggestion
*.exe.manifest
```
let's use glob so we can ignore all other `.exe` manifest.